### PR TITLE
Issue 25547 not allow un publish page with running experiment

### PR DESCRIPTION
**Pull request by freddyucv**
### Proposed Changes
* Check f the Contentlet that you are trying to unpublish is a Page then if it has any Running Experiment throw a Exception

### CODE REVIEW

File: dotCMS/src/main/java/com/dotcms/content/elasticsearch/business/ESContentletAPIImpl.java [hunk 1]
```diff
@@ -3993,6 +3995,17 @@ private void internalUnpublish(final Contentlet contentlet, final User user, fin
 
         this.canLock(contentlet, user);
 
+        if (contentlet.isHTMLPage()) {
+            final Optional<Experiment> runningExperiment = APILocator.getExperimentsAPI().getRunningExperiments().stream()
```
Comment (jdotcms) [inline dotCMS/src/main/java/com/dotcms/content/elasticsearch/business/ESContentletAPIImpl.java:3999]: shouldn't be a method such as getExperimentPerPage

Comment (freddyucv) [reply dotCMS/src/main/java/com/dotcms/content/elasticsearch/business/ESContentletAPIImpl.java]: done https://github.com/dotCMS/core/pull/25760/files#diff-8bf1865184fd59c5c64882df72bd0c71a403a544777830acf250bbf54ea84d90R238

File: dotCMS/src/main/java/com/dotcms/content/elasticsearch/business/RunningExperimentUnpublishException.java [hunk 2]
```diff
@@ -0,0 +1,15 @@
+package com.dotcms.content.elasticsearch.business;
+
+import com.dotcms.experiments.model.Experiment;
+import com.dotmarketing.exception.DotRuntimeException;
+import com.dotmarketing.portlets.contentlet.model.Contentlet;
+import com.dotmarketing.portlets.htmlpageasset.model.HTMLPageAsset;
+
+public class RunningExperimentUnpublishException extends DotRuntimeException {
```
[the remark this answers is not in the document]
Comment (freddyucv) [reply dotCMS/src/main/java/com/dotcms/content/elasticsearch/business/RunningExperimentUnpublishException.java:11]: done https://github.com/dotCMS/core/pull/25760/files#diff-31d5224c2efe751ae5b1b2adefc05140cfec64fb4bd60d4ac9a87150e8de7320R9

**Review by AndreyDotcms — CHANGES_REQUESTED** [review]

File: dotCMS/src/integration-test/java/com/dotcms/experiments/business/ExperimentAPIImpIntegrationTest.java [hunk 3]
```diff
@@ -6161,4 +6161,28 @@ public void renderUrlMapPageWithExperiment() throws DotDataException, DotSecurit
         }
     }
 
+    @Test
```
Comment (AndreyDotcms) [inline dotCMS/src/integration-test/java/com/dotcms/experiments/business/ExperimentAPIImpIntegrationTest.java:6164]: Please add remaining documentation

Comment (freddyucv) [reply dotCMS/src/integration-test/java/com/dotcms/experiments/business/ExperimentAPIImpIntegrationTest.java:6173]: done https://github.com/dotCMS/core/pull/25760/files#diff-34e7ad2b2aec5cebbf7ae6bb1697bff210c96c602184a9237bfda1a4c53e3c13R6164

**Review by AndreyDotcms — APPROVED** [review]

**Review by jdotcms — APPROVED** [review]

**Pull request merged**